src/ring_theory/algebra_tower.lean

```diff
@@ -262,8 +262,8 @@ variables {B}
 def alg_hom_equiv_sigma :
   (C →ₐ[A] D) ≃ Σ (f : B →ₐ[A] D), @alg_hom B C D _ _ _ _ f.to_ring_hom.to_algebra :=
 { to_fun := λ f, ⟨f.restrict_domain B, f.extend_scalars B⟩,
-  inv_fun := λ fg, @is_scalar_tower.restrict_base A _ _ _ _ _ _ _ _ _
-    fg.1.to_ring_hom.to_algebra _ _ _ _ fg.2,
+  inv_fun := λ fg,
+    let alg := fg.1.to_ring_hom.to_algebra in by exactI fg.2.restrict_base A,
```

Comment:
This one was fine with the underscores, but is much less fragile to argument reordering this way.